contracts/dao/neutron-staking-tracker/src/contract.rs

```diff
@@ -650,6 +650,12 @@ pub fn query_list_delegations(
 pub fn migrate(deps: DepsMut, _env: Env, _msg: MigrateMsg) -> Result<Response, ContractError> {
     // Set contract to version to latest
     set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
+
+    // Remove validator
+    let validator_addr = Addr::unchecked("neutronvaloper1v9xys5c4zdr89tvwq983ycnj3j4pekpjwr0raa");
+    let remove_height = 26438494;
+    VALIDATORS.remove(deps.storage, &validator_addr, remove_height)?;
```

Comment:
we remove the val by a hardcoded address here and remove only one specific validator. if by the time of the migration another validator(s) gets removed, we'll have more invalid validators stored in the contract's state, and this migration won't remove them. we'll have to perform another migration to clean up the store. do you see possible implementation ways that could consider that and would remove all such validators?

Reply:
We discussed this, and there's no way to handle it properly because of the snapshot map (but would love to hear your thoughts here).

The invalid validator doesn't make any harm to the contract, so in the worst case scenario we'll just roll out the fix but there will be some new invalid vals that we'll need to remove by another migration.